From 6b1037c7360264d83820168ac1b6adde26288d3d Mon Sep 17 00:00:00 2001
From: "Subhransu S. Prusty" <subhransu.s.prusty@intel.com>
Date: Fri, 21 Mar 2014 09:47:10 +0530
Subject: [PATCH 19/39] ASoC: soc-fw: Increase the fw text size to 44

Issue: GMINL-3679

As the control name may exceed 32 characters increase
the fw text size to 44.

Change-Id: If2ceddbbbcebadafaf1450cff584bb0314d76e64
Signed-off-by: Subhransu S. Prusty <subhransu.s.prusty@intel.com>
Signed-off-by: Mythri P K <mythri.p.k@intel.com>
---
 include/uapi/sound/asoc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/uapi/sound/asoc.h b/include/uapi/sound/asoc.h
index 56a0679..98519f6 100644
--- a/include/uapi/sound/asoc.h
+++ b/include/uapi/sound/asoc.h
@@ -762,7 +762,7 @@ enum snd_soc_dapm_type {
 #define SND_SOC_FW_MAGIC	0x41536F43 /* ASoC */
 
 /* string sizes */
-#define SND_SOC_FW_TEXT_SIZE	32
+#define SND_SOC_FW_TEXT_SIZE	44
 #define SND_SOC_FW_NUM_TEXTS	16
 
 /* ABI version */
-- 
1.9.1

